### DISCUSSION TITLE
GPIO Control plugin support

### DISCUSSION THREAD

**Discussion by idowantosell** [OctoFarm/OctoFarm#1318] [Feature Request]
Hi.
Would it possible to support the GPIO Control  plugin on the Octoprint ?

Thanks

BTW, I would use it to power on/off a LED light installed on the webcam
Ref: https://github.com/catgiggle/OctoPrint-GpioControl

**Comment by NotExpectedYet** [comment]
Hello, there is an API for it so it's certainly possible. 

I've been considering adding a light functionality similar to how the power one works. 

It's not a personal priority of mind but would accept a PR for it.

**Reply by idowantosell** [reply]
Thanks
